docs/DeveloperGuide.md

```diff
@@ -922,6 +922,27 @@ Currently, the process of modifying patient tags in the PatientSync application
 To address this, we are planning to introduce the EditTagsCommand feature, providing users with a more flexible and efficient way to manage patient tags. The planned EditTagsCommand feature is designed to enhance user productivity, reduce the likelihood of errors, and improve overall usability within PatientSync.
 
 
+### 4.3 Input Validation for Events
+
+Presently, the Date and Datetime for Events, referred to as `DATE_OR_DATETIME_OF_EVENT`, do not have sufficient input validation. For example, the user is able to input `30-02-2024, 24:00 - 24:00`. This can lead to potential errors if the user has accidentally mistyped the date and/or time when inputting the command, leading to confusion further down the line.
```

Comment:
Perhaps including the current input validation present, and planned specific input validation such as invalid time (eg. 24:00) or invalid dates (eg 31-02-2024) would be clearer!